src/Tizen.NUI/src/public/Common/Vector4.cs

```diff
@@ -718,7 +718,7 @@ public void Clamp(Vector4 min, Vector4 max)
 
         /// <inheritdoc/>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public object Clone() => new Vector4(X, Y, Z, W);
+        public object Clone() => ObjectPool.NewVector4(X, Y, Z, W);
```

Comment:
I think we cannot assume that `Clone()` API called only at main thread.

ObjectPool is not thread safe class. So we MUST not change like this.

Reply:
Done.